core/tests/protractor_desktop/navigation.js

```diff
@@ -214,6 +215,13 @@ describe('Static Pages Tour', function() {
       by.css('.protractor-test-thanks-page')).isPresent()).toBe(true);
   });
 
+  it('visits the Welcome Guide Page', function() {
+    browser.get('/welcome_guide');
+    waitFor.pageToFullyLoad();
```

Comment:
As discussed in https://github.com/oppia/oppia/pull/9247#discussion_r420500615, this doesn't work unless there's a loading symbol displayed while the page is loading. You need to wait for an element to appear instead

Reply:
fixed this